README.md

```diff
@@ -1,15 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 # HelloMicroverse
 
-The main goals of this project are:
-The main goals of this project to understand these things: 
--How to create branches with different base branches.
-1-->To deal with version control.
-2-->Follow GitHub flow.
-3-->Open pull request.
-4-->Write descriptive to understand commit messages.
-5-->Write short, detailed and easy to understand descriptions of pull requests.
-6-->Demonstrate an ability to submit a project for code review.
+
+>The main goals of this project are:
+ - How to create branches with different base branches.
+ - To deal with version control.
+ - Follow GitHub flow.
+ - Open pull request.
+ - Write descriptive to understand commit messages.
+ - Write short, detailed and easy to understand descriptions of pull requests.
+ - Demonstrate an ability to submit a project for code review.
+# In this project,3 files are added,
+
+  - A .html file that includes a header with the text "Hello Microverse!".
+
+  - A .css file that includes one class with styles for your header.
+
+  - A test.md file with the text "This file should be ignored by git".
```

Comment:
- [ ] Great job so far 👏. However, as the previous reviewer mentioned, in order for your README file to look professional, you need to add the `Getting started` section where you describe the procedures to clone and use your repository locally. T**his helps your fellow developers to know how they can use your project and this is great for development 😃 so please replace these highlighted texts and add the Getting started section**

Reply:
I mentioned everything:-
on headings, i just mentioned what is the goal of the projects and after that, I wrote what is in the projects.

Reply:
so i have to remove this line: In this project,3 files are added.?

Reply:
I have just the last review option left plz tell me where I am going wrong.